protos/flyteidl/admin/workflow.proto

```diff
@@ -65,3 +65,21 @@ message WorkflowClosure {
     // Time at which the workflow was created.
     google.protobuf.Timestamp created_at           = 2;
 }
+
+// The workflow id is already used and the structure is different
+message WorkflowErrorExistsDifferentStructure {
+    core.Identifier id = 1;
```

Comment:
Should we include the structure of the other workflow?

Reply:
Current plan was to return the id of the workflow to the sdk client and then to get the workflow and as part of the exception handling to then compare the diff with the initial request